build.gradle

```diff
@@ -67,20 +55,6 @@ dependencies {
     api "com.exactpro.th2:grpc-service-generator:${serviceGeneratorVersion}"
 }
 
-sourceSets {
-    main {
-        proto {
-            srcDirs 'src/main/proto'
-        }
-        java {
-            srcDirs 'src/gen/main/java', 'src/gen/main/grpc', 'src/gen/main/services/java/src'
-        }
-        resources {
-            srcDirs 'src/gen/main/services/java/resources'
-        }
-    }
-}
-
 protobuf {
```

Comment:
We don't need this
`generateProtoTasks.generatedFilesBaseDir = genDir`
the updated version of protobuf plugin puts the generated source to `build/generated` and add them as source set.
If we leave this property here it causes files duplication and you had to fix that (line 111)